composer.json

```
@@ -7,10 +7,10 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "silverstripe/recipe-plugin": "^1.2",
-        "silverstripe/assets": "1.x-dev",
-        "silverstripe/config": "1.x-dev",
-        "silverstripe/framework": "4.x-dev",
-        "silverstripe/mimevalidator": "2.x-dev"
+        "silverstripe/assets": "1.11.x-dev",
+        "silverstripe/config": "1.11.x-dev",
```

```suggestion
        "silverstripe/config": "1.4.x-dev",
```
That's what I get for trusting my brain instead of checking when I approved it

---

Updated